lua/grapple-line.lua

```diff
@@ -27,9 +27,10 @@ end
 
 local function make_statusline(files)
 	local result = {}
+	local match_color = M.settings.active_highlight or "lualine_a_normal"
```

Comment:
Thanks for the PR! That's a great suggestion. Would it be possbile to move that default value into the settings table? And it also might be good to add configuration for the inactive color too. What do you think about something in the settings table that looks like this?
```
colors = {
        active = 'lualine_a_normal',
        inactive = 'lualine_a_inactive',
      },
```